src/js/tech/html5.js

```diff
@@ -321,7 +321,7 @@ class Html5 extends Tech {
     if ('webkitDisplayingFullscreen' in video) {
       this.one('webkitbeginfullscreen', function() {
         this.one('webkitendfullscreen', function() {
-          this.trigger('fullscreenchange');
+          this.trigger('fullscreenchange', { isFullscreen: false });
         });
 
         this.trigger('fullscreenchange');
```

Comment:
Thanks for adding it. I don't really understand why it was working otherwise, but feels better to be consistent.